demos/usecases/ui/app.slint

```diff
@@ -42,4 +42,6 @@ export component App inherits Window {
     changed height => {
         DialogGlobal.window-height = root.height;
     }
+
+    out property <string> trick-to-include-german-umlauts: "äöüÄÖÜ"; // required for translations
```

Comment:
This sucks 😞 
We need a better way to tell the Slint compiler what ranges we include.
Alternatively, we should have the slint compiler look at the bundled string.
Anyway, I guess right now we have to keep that for the example.